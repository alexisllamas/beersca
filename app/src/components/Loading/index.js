import React from 'react';
import Icon from 'react-fontawesome';

const Loading = () => (
  <div>
    <Icon name="spinner" size="3x" spin />
    <span className="sr-only">Loading...</span>
  </div>
);

export default Loading;
