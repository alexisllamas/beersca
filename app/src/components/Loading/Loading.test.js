import React from 'react';
import mount from 'enzyme/mount';
import Loading from '.';

describe('Loading component', () => {
  it('should render correctly', () => {
    const wrapper = mount(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
