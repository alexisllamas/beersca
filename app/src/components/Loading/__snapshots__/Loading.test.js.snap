// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render correctly 1`] = `
<Loading>
  <div>
    <FontAwesome
      name="spinner"
      size="3x"
      spin={true}
    >
      <span
        aria-hidden={true}
        className="fa fa-spinner fa-3x fa-spin"
      />
    </FontAwesome>
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</Loading>
`;
